### DISCUSSION TITLE
Integration with Authentik

### DISCUSSION THREAD

**Discussion by kaygdotorg** [umputun/remark42#1739] [Ideas]
Hey, is it possible to integrate with Authentik instead of each provider separately?

**Comment by paskal** [comment]
[Here](https://goauthentik.io/docs/core/architecture) is the Authentik architecture. To support it, someone needs to figure out how to connect Remark42 to it on the backend and frontend side of things. I presume it's doable, but I found no documentation on that matter yet to evaluate the complexity of this task; please share it if you find it.
